packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -38,7 +38,7 @@ const Div100vhContainer = ({
         height: max_autoheight_offset ? '' : height_rule,
         maxHeight: max_autoheight_offset ? `calc(100rvh - ${max_autoheight_offset})` : '',
     };
-    if (is_bypassed) return children;
+    if (is_bypassed) return <React.Fragment>{children}</React.Fragment>;
```

Comment:
Could you please run all test on your machine?
In my previous PR's using React.Frament in such cases led to errors.

Reply:
No failed testcases. But I have explicitly typecased children to JSX.Element as suggested by @maryia